host/4/bullseye/amd64/dotnet/dotnet-inproc/dotnet.Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the runtime from source
-ARG HOST_VERSION=4.1.2
+ARG HOST_VERSION=4.1.3
 FROM mcr.microsoft.com/dotnet/sdk:6.0.100-rc.2 AS runtime-image
```

Comment:
@kshyju  / @CooperLink  - not a blocker , is this pointing to right image? still says rc2

Reply:
These are the old files. With cooper's refactoring, I do not think we need to update these anymore. We should delete these files to avoid confusion. @CooperLink could you confirm?

Reply:
Non-appservice image. Jumping into this branch to clean up all rc2 instances

Reply:
Wasn't able to make it to this branch before release. Bringing into dev before next release.